device_test.go

```diff
@@ -193,19 +193,9 @@ func TestInterfaceDescriptionError(t *testing.T) {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
-			c := newContextWithImpl(newFakeLibusb())
-			defer func() {
-				if err := c.Close(); err != nil {
-					t.Errorf("Context.Close(): %v", err)
-				}
-			}()
-			dev, err := c.OpenDeviceWithVIDPID(0x8888, 0x0002)
-			if dev == nil {
-				t.Fatal("OpenDeviceWithVIDPID(0x8888, 0x0002): got nil device, need non-nil")
-			}
-			defer dev.Close()
+			dev, err := createFakeDevice(0x8888, 0x0002)
 			if err != nil {
-				t.Fatalf("OpenDeviceWithVIDPID(0x8888, 0x0002): %v", err)
+				t.Error(err)
```

Comment:
that would need to remain t.Fatal rather than t.Error - the test is meaningless past this point.